docs/models/operations/partitionresponse.md

```diff
@@ -8,4 +8,5 @@
 | `content_type`                                               | *str*                                                        | :heavy_check_mark:                                           | HTTP response content type for this operation                |
 | `status_code`                                                | *int*                                                        | :heavy_check_mark:                                           | HTTP response status code for this operation                 |
 | `raw_response`                                               | [httpx.Response](https://www.python-httpx.org/api/#response) | :heavy_check_mark:                                           | Raw HTTP response; suitable for custom response parsing      |
+| `csv_elements`                                               | *Optional[str]*                                              | :heavy_minus_sign:                                           | Successful Response                                          |
 | `elements`                                                   | List[Dict[str, *Any*]]                                       | :heavy_minus_sign:                                           | Successful Response                                          |
```

Comment:
TODO: This isn't displayed as optional, but `response.elements` is `None` when `csv_elements` is returned.